### ISSUE TITLE
🔗 🐞 Nested accordion is really slow (Duplicate of #24076)

### ISSUE THREAD

**Issue by Jerostel** [elementor/elementor#24960]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Nested Accordion is really slow when you try to duplicate, edit, or want to publish.

### Steps to reproduce

1. activate nesting and containers
2. add an accordion with tekst elements or more in the items
3. add like / for example 20 items to the accordion
4. try to duplicate and publish
5. you see it takes really long and it's faltering

### Expected behavior

1. duplicate item
2. smooth within 1 second
3. publish within 2 seconds

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.24.0
	MySQL version: Ubuntu 22.04 v10.6.16
	PHP Version: 8.2.15
	PHP Memory Limit: 1200M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 24M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.2
	Site URL: https://zonzo.dev.impression.nl
	Home URL: https://zonzo.dev.impression.nl
	WP Multisite: No
	Max Upload Size: 24 MB
	Memory limit: 40M
	Max Memory limit: 1200M
	Permalink Structure: /%postname%/
	Language: nl-NL
	Timezone: Europe/Amsterdam
	Debug Mode: Inactive

== Theme ==
	Name: impression ® | child theme
	Version: 1.0.0
	Author: impression ® | creative design agency
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.8.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: nl_NL
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

== Active Plugins ==
	Admin and Site Enhancements
		Version: 5.7.1
		Author: Bowo

	Elementor
		Version: 3.18.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.18.3
		Author: Elementor.com

	JetEngine
		Version: 3.3.1
		Author: Crocoblock

	JetEngine - Get attachment file link by ID
		Version: 1.1.3
		Author: Crocoblock

	Members
		Version: 3.2.9
		Author: MemberPress

	Rank Math SEO
		Version: 1.0.211
		Author: Rank Math

	Skyboot Custom Icons for Elementor - Elementor Icons library
		Version: 1.0.7
		Author: Skybootstrap

	SVG Support
		Version: 2.5.5
		Author: Benbodhi

	UiPress
		Version: 2.4.4
		Author: Admin 2020


== Gebruik van elementen ==
	
	archive : 1
		button : 1
		container : 5
		heading : 2
		jet-listing-grid : 1
		shortcode : 1
		text-editor : 1
	footer : 1
		container : 9
		divider : 2
		heading : 2
		icon-list : 1
		image : 1
		jet-listing-grid : 2
		social-icons : 1
		text-editor : 1
	header : 1
		button : 2
		container : 11
		icon : 1
		icon-list : 1
		image : 1
		jet-listing-grid : 4
		mega-menu : 1
		social-icons : 1
	page : 2
		button : 18
		container : 14
		heading : 6
		html : 1
		icon : 2
		icon-box : 14
		image : 1
		text-editor : 2
	popup : 3
		button : 1
		container : 6
		form : 1
		heading : 2
		icon-list : 1
		image-box : 1
		jet-listing-grid : 3
		nested-accordion : 2
		social-icons : 1
		text-editor : 1
		theme-site-logo : 1
	widget : 0
		button : 5
		form : 2
		image-carousel : 1
	wp-post : 0
		button : 9
		container : 12
		heading : 3
		html : 1
		icon : 1
		icon-box : 7
		text-editor : 1
	wp-page : 18
		button : 24
		container : 404
		divider : 23
		form : 1
		google_maps : 1
		heading : 88
		icon : 20
		icon-box : 4
		icon-list : 11
		image : 78
		image-carousel : 2
		jet-listing-grid : 8
		nested-accordion : 1
		nested-tabs : 1
		text-editor : 116
	single-post : 2
		container : 12
		gallery : 1
		heading : 5
		icon-list : 1
		jet-listing-dynamic-field : 1
		loop-grid : 1
		shortcode : 2
		text-editor : 1
		theme-post-content : 1
		video : 1
	single-page : 4
		button : 6
		container : 71
		divider : 4
		heading : 22
		icon : 3
		icon-list : 1
		image : 8
		jet-listing-dynamic-field : 2
		jet-listing-grid : 4
		nested-tabs : 1
		shortcode : 3
		text-editor : 22
		theme-post-content : 2
	loop-item : 2
		container : 4
		heading : 1
		icon-list : 1
		image : 1
		text-editor : 1
	jet-listing-items : 0
		button : 1
		call-to-action : 2
		container : 18
		heading : 6
		icon : 5
		icon-list : 1
		image : 4
		jet-listing-dynamic-field : 1
		jet-listing-dynamic-terms : 1
		text-editor : 3
		theme-post-excerpt : 1


== Instellingen ==
	
	cpt_support: post, page, uip-admin-page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	disabled_elements: nested-carousel, progress-tracker, stripe-button, paypal-button, video-playlist, code-highlight, lottie, template, facebook-page, facebook-embed, facebook-comments, facebook-button, blockquote, sitemap, post-info, post-navigation, post-comments, author-box, search-form, share-buttons, countdown, table-of-contents, reviews, testimonial-carousel, media-carousel, flip-box, price-table, price-list, hotspot, animated-headline, nav-menu, login, slides, portfolio, posts, loop-carousel, archive-posts, theme-archive-title, theme-post-featured-image, theme-post-title, theme-page-title, theme-site-title, text-path, jet-listing-dynamic-repeater, jet-listing-dynamic-meta, jet-listing-dynamic-link, jet-listing-dynamic-image, wp-widget-members-widget-users, wp-widget-members-widget-login, wp-widget-block, wp-widget-custom_html, wp-widget-nav_menu, wp-widget-tag_cloud, wp-widget-rss, wp-widget-recent-comments, wp-widget-recent-posts, wp-widget-categories, wp-widget-text, wp-widget-search, wp-widget-meta, wp-widget-media_video, wp-widget-media_gallery, wp-widget-media_image, wp-widget-media_audio, wp-widget-archives, wp-widget-calendar, wp-widget-pages, rating, read-more, sidebar, menu-anchor, audio, alert, testimonial, progress, counter, spacer, inner-section, image-gallery


== Eigenschappen ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor experimenten ==
	Geoptimaliseerde DOM-uitvoer: Actief
	Verbeterd laden van assets: Standaard actief
	Verbeterd laden van CSS: Actief
	Pictogrammen inline lettertype: Standaard inactief
	Extra aangepaste onderbrekingspunten: Standaard actief
	admin_menu_rearrangement: Standaard inactief
	Flexbox container: Actief
	Upgrade Swiper bibliotheek: Standaard actief
	Raster container: Standaard inactief
	Default to New Theme Builder: Standaard actief
	Hello Theme header &amp; footer: Standaard actief
	Editor top balk: Actief
	Geoptimaliseerd laden van Gutenberg: Standaard actief
	Bouwen met AI: Standaard inactief
	Landingspagina&#039;s: Standaard actief
	Geneste elementen: Actief
	Lazy-load achtergrondafbeeldingen: Standaard inactief
	Afbeelding laden optimaliseren: Standaard actief
	Pagina&#039;s paneel: Standaard inactief
	Globale stijlgids: Actief
	Page Transitions: Standaard actief
	Notes: Standaard actief
	Form Submissions: Standaard actief
	Scroll Snap: Standaard actief
	Menu: Actief
	Taxonomy Filter: Standaard inactief


**== Log ==
	
Log: showing 19 of 192023-12-10 02:46:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-12-10 02:46:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-12-10 02:46:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.0',
  'to' => '3.18.1',
)]
2023-12-10 02:46:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.0',
  'to' => '3.18.1',
)]
2023-12-21 02:48:17 [info] elementor::elementor_updater Started 
2023-12-21 02:48:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-12-21 02:48:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.18.2',
  'to' => '3.18.3',
)]
2023-12-21 02:48:18 [info] Elementor/Upgrades - _on_each_version Finished 
2023-12-21 02:48:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.18.2',
  'to' => '3.18.3',
)]
2023-12-22 02:46:12 [info] elementor-pro::elementor_pro_updater Started 
2023-12-22 02:46:12 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-12-22 02:46:12 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-12-22 02:46:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.1',
  'to' => '3.18.2',
)]
2023-12-22 02:46:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.1',
  'to' => '3.18.2',
)]
2024-01-21 02:46:56 [info] elementor-pro::elementor_pro_updater Started 
2024-01-21 02:46:56 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-01-21 02:46:56 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-01-21 02:46:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.2',
  'to' => '3.18.3',
)]
2024-01-21 02:46:56 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.2',
  'to' => '3.18.3',
)]

PHP: showing 9 of 9PHP: 2023-12-11 16:58:41 [notice X 293][../wp-content/plugins/elementor/includes/base/controls-stack.php::228] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-12-12 12:00:37 [warning X 4][../wp-content/plugins/elementor/includes/base/controls-stack.php::1459] Undefined array key &quot;_flex_size_laptop&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-12-12 12:04:59 [notice X 2304][../wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-22 14:36:45 [notice X 1][../wp-content/plugins/elementor-pro/core/utils.php::383] Use of &quot;self&quot; in callables is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-22 16:14:49 [notice X 2][../wp-content/plugins/elementor-pro/modules/loop-builder/files/css/loop.php::57] Creation of dynamic property ElementorPro\Modules\LoopBuilder\Files\Css\Loop::$post_id is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-22 16:14:51 [warning X 4][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::179] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-23 13:57:17 [warning X 1][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-23 14:06:06 [notice X 6][../wp-content/plugins/elementor/includes/utils.php::727] strtolower(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/includes/utils.php(727): class type strtolower()
#2: ../wp-content/plugins/elementor-pro/modules/call-to-action/widgets/call-to-action.php(1594): Elementor\Utils :: validate_html_tag()
#3: ../wp-content/plugins/elementor/includes/base/controls-stack.php(2297): ElementorPro\Modules\CallToAction\Widgets\Call_To_Action -> render()
#4: ../wp-content/plugins/elementor/includes/base/widget-base.php(614): Elementor\Controls_Stack -> render_by_mode()
',
)]
PHP: 2024-01-26 18:53:52 [notice X 14][../wp-content/plugins/elementor-pro/modules/custom-css/module.php::96] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 14 of 14JS: 2023-12-11 16:33:56 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.18.2:3:690171] this.model.isValidChild is not a function 
JS: 2024-01-16 09:50:07 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view3967&quot;) has already been destroyed and cannot be used. 
JS: 2024-01-18 13:51:23 [error X 13][../wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.18.3:2:56156] Model.isValidChild() should be implemented, please provide &#039;isValidChild&#039; functionality. 
JS: 2024-01-19 07:28:05 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.18.3:3:601974] Cannot read properties of null (reading &#039;id&#039;) 
JS: 2024-01-19 15:47:44 [error X 10][../wp-admin/load-scripts.php?c=0&amp;loadchunk_0=jquery-core,jquery-migrate,utils,moxiejs,plupload&amp;ver=6.4.2:2:28760] Cannot read properties of undefined (reading &#039;value&#039;) 
JS: 2024-01-22 13:03:13 [error X 12][../wp-includes/js/dist/vendor/react-dom.min.js?ver=18.2.0:10:23755] Commands: `panel/global/open` not found. 
JS: 2024-01-22 13:09:09 [error X 2][../wp-includes/js/dist/vendor/react-dom.min.js?ver=18.2.0:10:23755] this.manager.setPage(...).showView is not a function 
JS: 2024-01-23 08:05:24 [error X 1][../wp-content/plugins/elementor/assets/js/editor-document.min.js?ver=3.18.3:2:1122] container or containers are required. 
JS: 2024-01-23 13:49:41 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.18.3:3:806339] Cannot read properties of undefined (reading &#039;toLowerCase&#039;) 
JS: 2024-01-24 07:28:24 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view8824&quot;) has already been destroyed and cannot be used. 
JS: 2024-01-25 13:16:12 [error X 2][../wp-content/plugins/elementor/assets/js/4fdaa70e951ad90db2f2.bundle.min.js:2:7786] Cannot read properties of null (reading &#039;dataset&#039;) 
JS: 2024-01-26 10:41:05 [error X 2][../wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.18.3:2:16880] Cannot read properties of undefined (reading &#039;attributes&#039;) 
JS: 2024-01-26 17:53:44 [error X 1][../wp-content/plugins/elementor/assets/js/packages/editor-documents/editor-documents.min.js?ver=3.18.3:1:4405] Cannot read properties of null (reading &#039;editor&#039;) 
JS: 2024-01-26 17:53:49 [error X 1][../wp-content/plugins/elementor/assets/js/4fdaa70e951ad90db2f2.bundle.min.js:2:7998] Cannot read properties of undefined (reading &#039;view&#039;)  



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibel
	Skyboot Custom Icons for Elementor - Elementor Icons library: Compatibiliteit niet gespecificeerd

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by samuelmente** [comment]
I think this is a duplicate of #24076.
But it should really be adressed soon..

**Comment by nicholaszein** [comment]
Hello @Jerostel!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #24076 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.23.0! 🥳 

The time to interaction should improve by 60%-98%, and performance overall should be better. For a full breakdown please read [this article](https://developers.elementor.com/elementor-3-23-developers-update/#:~:text=Performant%20Nested%20Elements%20In%20the%20Editor).

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
